tests/python/tank_test/tank_test_base.py

```diff
@@ -212,12 +216,9 @@ def setUp(self, parameters=None):
         mockgun.Shotgun.set_schema_paths(mockgun_schema_path, mockgun_schema_entity_path)
 
         self.tank_temp = TANK_TEMP
-        self.init_cache_location = os.path.join(self.tank_temp, "init_cache.cache")
```

Comment:
There's no need to override that path anymore. We might as well retrieve the correct value from the pipeline config factory.